clients/drcachesim/drpt2trace/pt2ir.h

```diff
@@ -342,66 +323,61 @@ class pt2ir_t {
     ~pt2ir_t();
 
     /**
-     * Returns true if the pt2ir_t is successfully initialized. Returns false on failure.
-     * \note Parse struct pt2ir_config_t and initialize PT instruction decoder, the
-     * sideband session, and images caches.
+     * Returns true if the instance is successfully initialized. Returns false on failure.
+     * \note Parse struct #pt2ir_config_t and initialize the PT instruction decoder, the
+     * sideband session.
```

Comment:
Document the meaning of the new args (init_pt_data, etc).

Reply:
Done.